docs: update contrast.mdx (#3999)
digdir/designsystemet: apps/www/app/content/best-practices/en/accessibility/contrast.mdx

      **WCAG 2.2: 2.4.13** Focus Appearance (Level AAA), the appearance of
      the focus indicator requires that contrast ratio between the same 
      pixels in the focused and unfocused states must be at least 3:1.
      [Understanding Success Criterion 2.4.13: Focus Appearance (w3.org)](https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance.html).
    </li>
    <li>
      **WCAG 3** discuss a requirement for color and contrast, visual contrast in text
      (silver): Ensure sufficient contrast between foreground text and the
      background of the text. A new method, called APCA,
      is suggested to calculate the contrast.
    </li>
  </ul>
</Card>

## A new and more accurate method

> In WCAG 3.0 there is discussions around a more accurate method than today's standard for calculating contrast and setting threshold values.

Reviewer: ```suggestion
In WCAG 3.0 there are discussions around a more accurate method than today's standard for calculating contrast and setting threshold values.
```

To ensure that the content in public digital services is accessible to all users, we aim to achieve compliance with WCAG 2.1 AAA.

> To further enhance readability, we have ambitions to achieve contrast at level 4. This is the top level conformance, which is being discussed in a for now [inactive work on contrast in WCAG 3.0 (w3.org)](https://www.w3.org/WAI/GL/task-forces/silver/wiki/Visual_Contrast_of_Text_Subgroup)

Reviewer: ```suggestion
To further enhance readability, we have ambitions to achieve contrast at level 4. This is the top level conformance, which is being discussed in an [inactive proposal on contrast in WCAG 3.0 (w3.org)](https://www.w3.org/WAI/GL/task-forces/silver/wiki/Visual_Contrast_of_Text_Subgroup)
```

In variant A, the text has the color #6D7879 with a white background. The text is somewhat readable.
In variant B, the text is black, and the background has the color #6D7879. The text is slightly less readable than variant A.

## References and further discussions

> Severall discussions in the ongoing work on WCAG 3 is on the tpoic of contrast. There is also wikis from both active and inactive working subgroups that can shed light on the situation.

Reviewer: ```suggestion
Several discussions in the ongoing work on WCAG 3 is on the topic of contrast. There is also wikis from both active and inactive working subgroups that can shed light on the situation.
```

- Visual Contrast of Text Subgroup, an inactive group working on contrast, [suggested that conformance for AAA-level in WCAG 2 would be equal to Score 3 in what was then the discussed conformance-levels of WCAG 3 (w3.org)](https://www.w3.org/WAI/GL/task-forces/silver/wiki/Visual_Contrast_of_Text_Subgroup#)

> - The published draft as of august 2025, for [WCAG 3 does not include the contrast requirements or the method for APCA method. (w3.org)](https://www.w3.org/TR/wcag-3.0/#maximum-text-contrast)

Reviewer: ```suggestion
- The published draft as of August 2025, for [WCAG 3 does not include the contrast requirements or the method for APCA method. (w3.org)](https://www.w3.org/TR/wcag-3.0/#maximum-text-contrast)
```

> - Multiple [reference tables have been suggested used for various conformance levels (w3.org)](https://www.w3.org/WAI/GL/task-forces/silver/wiki/User:Myndex/Conformance), some of them suggested by the author of APCA can provide an understanding of the complexity.

Reviewer: ```suggestion
- Multiple [reference tables have been suggested for various conformance levels (w3.org)](https://www.w3.org/WAI/GL/task-forces/silver/wiki/User:Myndex/Conformance), some of them suggested by the author of APCA can provide an understanding of the complexity.
```

- [Myndex Github repo for w3c WCAG 3 (github.com)](https://github.com/Myndex/apca-w3)
- Find [multiple discussions in the WCAG 3 Github repo on the subject of contrast (github.com)](https://github.com/w3c/wcag3/issues?q=is%3Aissue%20state%3Aopen%20contrast%20label%3A%22Path%3A%20Visual%20appearance%22), i.e. there is a [list of independent sources that has made their own investigations on APCA. (github.com)](https://github.com/w3c/wcag3/issues/29)

> - Interview with the creator of APCA, where he talk about subject like why the WCAG 2 contrast method is problematic and also about the science behind APCA, all with examples. [Video: How APCA Changes Accessible Contrast with Andrew Somers (youtube.com).](https://www.youtube.com/watch?v=E-CZial5VkU)

Reviewer: ```suggestion
- Interview with the creator of APCA, where he talks about subjects like "why the WCAG 2 contrast method is problematic" and also about the science behind APCA, all with examples. [Video: How APCA Changes Accessible Contrast with Andrew Somers (youtube.com).](https://www.youtube.com/watch?v=E-CZial5VkU)
```
